apps/dokuha-roadmap-frontend/src/app/details/[contentId]/page.tsx

```diff
@@ -0,0 +1,182 @@
+'use client';
+
+import { useParams, useRouter } from 'next/navigation';
+import { useEffect, useState } from 'react';
+
+import Button from "@/components/ui/Button";
+
+export type postData = {
+  currentPage: number;
+  note: string;
+}
+
+export type getData = {
+  userId: string;
+  title: string;
+  totalPage: number;
+  currentPage: number;
+  note: string;
+}
+
+//TODO:alertの箇所はトースト通知ないしはonBlurでやりたい
+export default function DetailPage() {
+  const router = useRouter();
+
+  const { contentId } = useParams();
+  const [contentTitle, setContentTitle] = useState('');
+  const [currentPage, setCurrentPage] = useState(0);
+  const [totalPage, setTotalPage] = useState(0);
+  const [note, setNote] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  const progress = (currentPage / totalPage) * 100;
```

Comment:
[IMO]
currentPage と totalPage の値が万が一にでも両方 0 だった時に NaN になるけど、それ自体はこのページの関心ではない（サーバーサイドでどうデータを持つかの問題）なので、ここは許容する形にすると良さげ

というのと、totalPageが 0 になることはないはず（これはドメイン的に）だと思うので、上記の 0 / 0 = NaN を防ぐ意味でも初期値は 1 の方が良さそう！
```suggestion
  const [currentPage, setCurrentPage] = useState(0);
  const [totalPage, setTotalPage] = useState(1);
  const [note, setNote] = useState('');
  const [isLoading, setIsLoading] = useState(true);

  const progress = totalPage > 0 ? (currentPage / totalPage) * 100 : 0;
```

Reply:
@k-naoyuki 
ここ直ってなさそう！